src/js/media-controller.js

```diff
@@ -325,6 +347,21 @@ class MediaController extends MediaContainer {
         }
         this.propagateMediaState(MediaUIAttributes.MEDIA_IS_PIP, isPip);
       },
+      // Note this relies on a customized video[is=castable-video] element.
+      'entercast,leavecast': (e) => {
+        const isCast = this.media && globalThis.CastableVideo?.castElement === this.media;
+
+        // Prevent auto hiding controls while casting.
+        if (isCast && this.autohide >= 0) {
+          this._prevAutohide = this.autohide;
+          this.setAttribute('autohide', '-1');
```

Comment:
blocker: Our pre-existing convention is to handle this stuff in media-container's code for activity monitoring & related css, since `autohide` may be set by implementors. See, e.g. the way we handle checking for paused state.

Reply:
good call, worked with just css in media-container